js/util/vectortile_to_geojson.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var VectorTileFeature = require('vector-tile').VectorTileFeature;
+
+module.exports = Feature;
```

Comment:
I have no strong preference either way. I don't feel weird about using `new` because it creates a js object that lazily generates geojson from a vectortile object that it holds a reference to. If it created a plain json-like object I'd feel weird.